Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -756,15 +756,15 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::Load(const u32 stage)
   }
 
   // how many levels the allocated texture shall have
-  const u32 texLevels = hires_tex ? (u32)hires_tex->m_levels.size() : tex_levels;
+  const u32 texLevels = custom_tex ? (u32)custom_tex->m_levels.size() : tex_levels;
```

Comment:
That change is fine and doesn't require a change of the PR title or commit message. I don't know if it's something I would require before the PR can get merged, though.